docs/reference.rst

```diff
@@ -0,0 +1,54 @@
+.. _reference_penaltymodel:
+
+Reference Documentation
+***********************
+
+.. currentmodule:: penaltymodel
+
+This package implements the generation and caching of :term:`penalty model`\ s.
+
+The main function for penalty models is:
```

Comment:
![image](https://user-images.githubusercontent.com/8395238/160443724-21631908-74a7-4dd8-8c82-01e13f6bbeff.png)

I think it looks fine and makes sense. Otherwise it would just link to the thing immediately below it.